packages/manager/src/features/CloudPulse/Alerts/CreateAlert/Criteria/MetricCriteria.tsx

```diff
@@ -66,16 +66,9 @@ export const MetricCriteriaField = (props: MetricCriteriaProps) => {
   });
 
   return (
-    <Box sx={(theme) => ({ marginTop: theme.spacing(3) })}>
-      <Box
-        alignItems="center"
-        display="flex"
-        justifyContent="space-between"
-        sx={{ marginBottom: 1 }}
-      >
-        <Typography variant="h2">3. Criteria</Typography>
-      </Box>
-      <Stack spacing={2} sx={(theme) => ({ marginTop: theme.spacing(3) })}>
+    <Stack spacing={2} sx={(theme) => ({ marginTop: theme.spacing(3) })}>
```

Comment:
We can just pass in `mt` for marginTop

```suggestion
    <Stack mt={3} spacing={2}>
```